.github/workflows/build.ci.yml

```diff
@@ -274,8 +274,6 @@ jobs:
             check
             jacocoTestReport
             koverXmlReport
-            :substrate:redakt:test
-            :substrate:koverXmlReport
```

Comment:
Removed tests and reports for the `substrate` project while it's detached from the build